doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -41,23 +41,22 @@ jupyter:
 
 ## Local connection to a Workbench Server
 
-Execute the steps below to link PyWorkbench with a local Ansys Workbench session:
+Execute the steps below to link PyWorkbench with a local session:
+
 - Initiate Ansys Workbench
 - Input StartServer() in the Workbench Command Window
 - Utilize the given port number to link PyWorkbench with the server
 
 ```{python}
 from ansys.workbench.core import launch_workbench
-
 wb = launch_workbench()
 ```
 
 ## Remote connection to a Workbench Server
 
-Execute the steps below to link PyWorkbench with a local Ansys Workbench session:
+Execute the steps below to link PyWorkbench with a local session:
```

Comment:
```suggestion
Perform these steps to link PyWorkbench with a remote session:
```
Shouldn't this be "with a remote session"?

Reply:
Done. Thanks!